callstats-java-sdk/src/main/java/io/callstats/sdk/httpclient/CallStatsHttpClient.java

```diff
@@ -160,12 +180,28 @@ public CallStatsHttpClient(CallStatsUrls url) {
 			throw new RuntimeException(e);
 		}
 
-		PoolingHttpClientConnectionManager mgr = new PoolingHttpClientConnectionManager();
-		mgr.setDefaultMaxPerRoute(10);
-		httpclient = HttpClients.custom().setConnectionManager(mgr).build();
-
 		PoolingNHttpClientConnectionManager asyncMgr = new PoolingNHttpClientConnectionManager(ioReactor);
-		httpAsyncClient = HttpAsyncClients.custom().setConnectionManager(asyncMgr).build();
+		
+		if (isProxyEnabled) {
+			logger.info("Proxy Enabled " + proxyHost +":"+ proxyPort);
+			HttpHost proxy = new HttpHost(proxyHost, proxyPort);
+	        config = RequestConfig.custom()
+	                .setProxy(proxy)
+	                .build();
+	        if (isProxySecure) {
+	        		credentialsProvider = new BasicCredentialsProvider();
+	        		credentialsProvider.setCredentials(new AuthScope(proxyHost, proxyPort),
+		                new UsernamePasswordCredentials(proxyUserName, proxyPassword));
+	        }
+		}
```

Comment:
The identation seems incorrect. Additionally I don't think we need to handle case where `proxyEnabled=false && isProxySecure=true `